src/test/java/features/Test Mathematical Steps.feature

```diff
@@ -0,0 +1,474 @@
+#language: en
+#Author: Tyler Bouchard
+
+@math
```

Comment:
There should be a ticket number for this. If there isn't one, make one in the sentinel project. This should be the tag, as well as at the front of the feature name and scenario names and the file name, just like any other test.

Reply:
Done! I added the issue number.